pages/_error.js

```diff
@@ -40,18 +44,17 @@ const Error = ({ statusCode = 0, fontsLoaded = '', wrapperClass = '' }) => (
 
 Error.getInitialProps = async ({ res, req, jsonPageRes }) => {
   const statusCode = res
-    ? res.statusCode
-    : jsonPageRes ? jsonPageRes.status : null
-  // temp fix for bug in nextjs rendering 500 page twice.
-  // keep an eye on this 'well explained' github issue for a possible fix
-  // https://github.com/zeit/next.js/issues/2964
-  const wrapperClass = statusCode === 404 ? '' : 'is-500-error'
+    ? res.statusCode.toString()
+    : jsonPageRes ? jsonPageRes.status.toString() : null
```

Comment:
I personally dislike this way of writing code. it abstracts away too much logic. maybe refactor this into a more readable code

Reply:
Since I only added the `.toString()` functions I will leave the rest of the code as is.

Reply:
should refactor in another ticket